package.json

```diff
@@ -1,23 +1,63 @@
 {
-  "name": "headers-generator",
-  "version": "0.0.1",
-  "description": "NodeJs package for generating browser-like headers.",
-  "main": "main.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "repository": {
+  "_args": [
```

Comment:
Could you please explain why is this here?

Reply:
Because I copied in the wrong package.json file and didn't notice. Will get rid of the ballast.